Log/README.md

```diff
@@ -62,6 +62,13 @@ metadata = {
 }
 ```
 
+- **Custom Tags** 
+
+You have two options to optionnally add custom tags to the logs:
```

Comment:
r/`You have two options to optionnally add custom tags to the logs:`/ `You have two options to optionally add custom tags to your logs:`

Reply:
@l0k0ms what about adding `(Optional)` in the title and then 
`You have two configuration possibilities to add custom tags to your logs:`